README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+
```

Comment:
For consistency with the rest of the repo, please don't have a leading newline between a heading and content